cypress/e2e/nav_spec.cy.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-undef */
+
+
+describe('nav', () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'https://everuse-be-b6017dbfcc94.herokuapp.com/graphql', {
+      statusCode: 201,
+      fixture: 'items.json'
+    }).as('getItems')
+    cy.visit('http://localhost:3000/')
```

Comment:
This can be replaced with the command `stubRequestsDynamically()`; using the command will also allow deletion of the items.json fixture.

Reply:
^ same

Reply:
Thanks for making this change. Tests are stubbing and passing perfectly.